src/hooks/loadables/useLoadSpendingLimits.ts

```diff
@@ -105,7 +105,7 @@ export const useLoadSpendingLimits = (): AsyncResult<SpendingLimitState[]> => {
     if (!provider || !safeLoaded || !safe.modules || !tokenInfoFromBalances) return
 
     return getSpendingLimits(provider, safe.modules, safeAddress, chainId, tokenInfoFromBalances)
-  }, [provider, safeLoaded, safe.modules?.length, safeAddress, chainId, safe.txHistoryTag, tokenInfoFromBalances])
```

Comment:
Are we sure the `txHistoryTag` needs to be removed here? I remember we wanted to automatically update the value on the settings page without reloading

Reply:
True, I understand the requirement. I've added it back.